page_objects/common_page.py

```diff
@@ -6,10 +6,11 @@ def __init__(self, driver=None):
         self.driver = driver
         self.connector_url = None
         self.modal_dialog = "//div[@class='f9-modal-dialog'][contains(., '<title>')]"
-        self.modal_submit_button = "//button[contains(@class, 'f9-positive-cta-btn') and contains(., '<text>')]"
+        self.modal_submit_button = "//button[contains(@class, 'f9-positive-cta-btn') and contains(., '<text>')] | //button[contains(@id, 'connect_station_btn') and contains(., '<text>')] | //button[contains(@id, 'confirm-softphone-setup') and contains(., '<text>')] | //button[contains(@id, 'skills_confirm_btn') and contains(., '<text>')]"
```

Comment:
does not the `confirm_selection_button` from `adapter_page.py` work?
maybe you can use the "When I confirm the...(skills, station)" steps. (let's discuss)

Reply:
You're right, I should use use the "When I confirm the...(skills, station)" steps.
Fixed